docs/hooks/index.md

```diff
@@ -35,64 +27,48 @@ An _Action_ defines one or more _Hooks_ to execute. lakeFS supports three types
 1. Notifying downstream consumers:
    Running a post-merge hook to trigger an Airflow DAG or to send a Webhook to an API, notifying it of the change that happened
 
-For step-by-step examples of hooks in action check out the [lakeFS samples repository](https://github.com/treeverse/lakeFS-samples/).
+For step-by-step examples of hooks in action check out the [lakeFS Quickstart](/quickstart/actions-and-hooks.html) and the [lakeFS samples repository](https://github.com/treeverse/lakeFS-samples/).
 
-## Supported Events
+## Overview
 
-| Event                | Description                                                                    |
-|----------------------|--------------------------------------------------------------------------------|
-| `pre-commit`         | Runs when the commit occurs, before the commit is finalized                    |
-| `post-commit`        | Runs after the commit is finalized                                             |
-| `pre-merge`          | Runs on the source branch when the merge occurs, before the merge is finalized |
-| `post-merge`         | Runs on the merge result, after the merge is finalized                         |
-| `pre-create-branch`  | Runs on the source branch prior to creating a new branch                       |
-| `post-create-branch` | Runs on the new branch after the branch was created                            |
-| `pre-delete-branch`  | Runs prior to deleting a branch                                                |
-| `post-delete-branch` | Runs after the branch was deleted                                              |
-| `pre-create-tag`     | Runs prior to creating a new tag                                               |
-| `post-create-tag`    | Runs after the tag was created                                                 |
-| `pre-delete-tag`     | Runs prior to deleting a tag                                                   |
-| `post-delete-tag`    | Runs after the tag was deleted                                                 |
+An _Action_ defines one or more _Hooks_ to execute. lakeFS supports three types of hook: 
 
-lakeFS Actions are handled per repository and cannot be shared between repositories.
-A failure of any Hook under any Action of a `pre-*` event will result in aborting the lakeFS operation that is taking place.
-Hook failures under any Action of a `post-*` event will not revert the operation.
+1. [Lua](./lua.html) - uses an embedded Lua VM
+1. [Webhook](./webhooks.html) - makes REST call to external URL and waits for response
+1. [Airflow](./airflow.html) - triggers a DAG in Airflow
```

Comment:
While we're here, I think that we can improve the explanation of synchronicity of calls.  So consider replacing everything about "waits for a response" and here.

Maybe we can have a separate paragraph such as

> "Before" hooks must run successfully before their action.  If the hook fails, it aborts the action.  Lua hooks and Webhooks are _synchronous_, and lakeFS waits for them to run to completion.  Airflow hooks are _asynchronous_: lakeFS stops waiting as soon as Airflow accepts triggering the DAG.

@johnnyaug WDYT?

Reply:
I like this; added.